samples/query-store/regressed-queries/README.md

```diff
@@ -0,0 +1,50 @@
+﻿# Identifying Regressed Queries with Azure Database for PostgreSQL Intelligent Performance
+ 
+## Introduction
+One of the challenges associated with monitoring and alerting is to filter out noise from the signal. Nobody wants to receive a midnight call by an automated alert for an issue that turns out to be a false positive. Nor do we want to lose hours sorting through issues to locate a starting point from which to triage. At any point, if you suffer from a slowdown in your workload, one way to digest the overall problem space is to review your top regressed queries using a dash of statistics. Doing so gives you a chance to rule out differences that are more likely to be “coincidental” so that you can concentrate on issues that are statistically significant. 
+
+In his blog post [Operationalizing your PostgreSQL database health checks using SQL Notebooks](https://azure.microsoft.com/en-us/blog/operationalizing-your-database-health-checks-using-sql-notebooks/), Parikshit Savjani describes a great way to [using notebooks in Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-notebooks?view=sql-server-2017) to operationalize PostgreSQL health checks. If you have started with basic health checks in your notebook, the exercise of extending your setup with the regressed queries scenario will rather be easy for you.
+ 
+## Getting started
+* Install [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/download?view=sql-server-2017)
+* Clone this repo or simply copy the notebook and the requirements.txt file into a local folder of your choice
+* Open the folder in Azure Data Studio [CTRL + K CTRL + O]
+* [Configure](https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-notebooks?view=sql-server-2017#manage-packages) local Python kernel for Notebooks by switching the kernel to Python 3.
+* Have an Azure Database for PostgreSQL instance configured as needed below.
+* Update required paramaters within the notebook per blow
+* Run your notebook
+
+## Configure your database
+You need to ensure below server settings for data collection to start.
+* go into Server parameters parameters blade on Azure portal
```

Comment:
You may want to consider using uppercase for the first letter.

Reply:
thanks @savjani ! incorporated your feedback